docs/appendices/release-notes/5.7.2.rst

```diff
@@ -52,6 +52,10 @@ Security Fixes
 Fixes
 =====
 
+- Fixed an issue leading to a `SQLParseException` when a scalar function was used
+  in the projection clause of a select statement in combination with a correlated
+  sub-query.
+
 - Fixed an issue that could lead to requests getting stuck when trying to
```

Comment:
I don't think we need a chanhe entry because it's a recebt regression.

>In 5.7.1 
the query in this ticket: works

In https://github.com/crate/crate/commit/3a9fd77cdeea1e40d018a3fd5317c6059a92bd66 (nightly-5.8.0-2024-05-23-00-02) - which got the fix for https://github.com/crate/crate/issues/15901
the query in this ticket: still works

Reply:
Ok, so just a backport with no entry?